_source/lib/apollos.coffee

```diff
@@ -37,10 +37,11 @@ Apollos.user.login = {}
 ###
 Apollos.person = ->
 
-  userDoc = Apollos.user()
+  user = Apollos.user()
 
-  person = Apollos.people.findOne
-    userId: userDoc._id
+  if user.rock?.guid
```

Comment:
@johnthepink any reason why we set this with a rock object?

Couldn't we do `Apollos.people.findOne(userGuid: user.guid)`

Then Rock could set that Guid on creation of Apollos people / user from Rock and Apollos could sync that data to Rock and set the Guid of the user / person.

Maybe I'm misunderstanding as well :crying_cat_face: